2025-01-19 06:45:57.572 | ERROR    | __main__:check_phone_numbers:231 - 检测失败: cannot access local variable 'index' where it is not associated with a value
